1-js/06-advanced-functions/06-function-object/article.md

```diff
@@ -3,7 +3,7 @@
 
 As we already know, a function in JavaScript is a value.
 
-Every value in JavaScript has a type. What type is a function?
+Every value in JavaScript has a type. What is a functions type?
```

Comment:
would recommend the below
```suggestion
Every value in JavaScript has a type. What is the type of a function?
```
<br>

if we are keeping the current one, then `functions` should be changed to `function's`